content/doc/book/system-administration/reverse-proxy-configuration-with-jenkins/reverse-proxy-configuration-pomerium.adoc

```diff
@@ -30,4 +30,269 @@ Once you’ve configured JWT authentication, you can assign permissions within J
+5. Select the JWT Auth plugin and **Install without restart**
+
+Install the link:https://plugins.jenkins.io/skip-certificate-check/[**skip-certificate-check** plugin]:
+
+1. Select **Available Plugins**
+2. In the search bar, enter **skip-certificate-check**
+3. Select the skip-certificate-check plugin and **Install without restart**
+
+Once you’ve installed both plugins, **stop your containers**.
+
+== Configure JWT authentication
+
+Go to your external Jenkins route.
+
+To configure JWT authentication:
+
+1. Go to **Manage Jenkins**
+2. Under **Security**, select **Configure Global Security**
+3. Under **Authentication** > **Security Realm**, select **JWT Header Authentication Plugin**
+
+Under **Global JWT Auth Settings**, you’ll see form fields where you can enter JWT claims. Pomerium forwards a user’s associated link:https://www.pomerium.com/docs/capabilities/getting-users-identity#jwt-verification[identity information] in a signed attestation JWT that’s included in upstream requests in an `X-Pomerium-Jwt-Assertion` header.
+
+With the JWT Auth plugin installed, Jenkins can receive and parse the assertion header to authenticate users – you just need to give it the right instructions to find the header and JWT claims.
+
+Enter the following information in the **Global JWT Auth Settings** field:
+
+
+.Global JWT Auth Settings
+|===
+|Field |Value
+
+|**Header name**
+|`x-pomerium-jwt-assertion`
+
+|**Username claim name**
+|`name` or `email`
+
+|**Groups claim name**
+|`groups`
+
+|**Groups claim list seperator**
+|`,`
+
+|**Email claim name** 
+|`email`
+
+|**Acceptable issuers**
+|`authenticate.corp.example.com`
+
+|**Acceptable audiences**
+|`jenkins.corp.example.com`
+
+|**JWKS JSON URL**
+|`https://jenkins.corp.example.com/.well-known/pomerium/jwks.json`
+|===
+
+Note the following details about the fields above:
+
+- **Username claim name** can be either your name or email
+- **Acceptable issuers** must be the URL of the authentication domain that issued the JWT. The `iss` claim tells the target application who the issuing authority is and provides context about the subject.
+- **Acceptable audiences** must be the URL of the target application. The `aud` claim defines what application the JWT is intended for.
+- **JWKS JSON URL** appends `/.well-known/pomerium/jwks.json` to the external route URL. The JWKS endpoint provides Jenkins the user’s public key to verify their JWT signature.
+
+You can go to the external `verify` route defined in your policy to view your JWT claims.
+
+In the **Authorization** dropdown, configure Jenkins permissions so that **Anonymous** has **Administer** privileges.
+
+1. Select **Matrix-based security**
+2. Under **Overall**, assign **Administer** to **Anonymous** and **Authenticated Users**
+
+If JWT authentication doesn't authenticate you successfully, Jenkins will sign you in as an anonymous user. With administer privileges, you can troubleshoot JWT settings as an anonymous user and try again.
```

Comment:
```suggestion
If JWT authentication doesn't authenticate you successfully, Jenkins signs you in as an anonymous user. With administer privileges, you can troubleshoot JWT settings as an anonymous user and try again.
```
Suggestion to use [active voice](https://learn.microsoft.com/en-us/style-guide/global-communications/writing-tips#tips-for-all-global-content) here instead of "will sign".